opensearch-operator/pkg/builders/dashboards.go

```diff
@@ -182,6 +182,8 @@ func NewDashboardsSvcForCr(cr *opsterv1.OpenSearchCluster) *corev1.Service {
 			Labels:    labels,
 		},
 		Spec: corev1.ServiceSpec{
+			// type will default to ClusterIP
+			Type: cr.Spec.Dashboards.Service.Type,
```

Comment:
looks fine but we need a default value, if the service.spec is empty we need that default